src/app/complex/complex-results/complex-navigator/table-structure/table-structure.component.ts

```diff
@@ -34,9 +43,10 @@ export class TableStructureComponent implements AfterViewInit {
   }
 
   ngAfterViewInit(): void {
-    const header = this.headerDiv.nativeElement;
-    this.setHorizontalShadowVisibility(header);
-    window.addEventListener('scroll', () => this.shadowTopVisible = header.getBoundingClientRect().top <= this.scrollStart());
+    if (!!this.headerDiv) {
+      setTimeout(() => this.setHorizontalShadowVisibility());
```

Comment:
This timeout fixes the error mentioned on the comment I left the other day. I've tried multiple things but nothing seemed to work except this.